packet/loadbalancers.go

```diff
@@ -41,6 +41,9 @@ func newLoadBalancers(client *packngo.Client, projectID, facility string, config
 	if len(cmparts) >= 2 {
 		configmapnamespace, configmapname = cmparts[0], cmparts[1]
 	}
+	if configmap == "none" {
```

Comment:
Our "none" or "disabled" doesn't follow the "namespace:configmapname" format, so we need to persist it after split above so that the loadbalancer reconciliation loop knows what to do.

Reply:
Got it. I actually find this confusing. We use the `disabled` as a user-facing flag, but at the entry point (not in `loadbalancers.go`), resolve it to the default name.

If this is meant to indicate, "don't use the configmap, but keep doing bgp stuff", then we probably should have a boolean flag for that, rather than overloading it. It makes sense to overload for the env var, as that is how the user indicates intent, but not so much for a value passed to a func, as there we have options.

Reply:
Happy with a cool bool.